fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonItemTags.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.item.Item;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.ItemTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonItemTags {
+	public static final TagKey<Item> PICKAXES = register("pickaxes");
+	public static final TagKey<Item> SHOVELS = register("shovels");
+	public static final TagKey<Item> HOES = register("hoes");
+	public static final TagKey<Item> AXES = register("axes");
+	public static final TagKey<Item> SHEARS = register("shears");
+	/**
+	 * For throwable weapons, like Minecraft tridents.
+	 */
```

Comment:
More comments should be added to group these out, its just a massive chunk after this currently

Reply:
Most tag names are self-explanatory

Reply:
Yes im more commenting on codebase clarity, this is a single (imo unnecessarily) multiline comment in the middle of a big chunk, they should really be separated visually or with comments to keep related ones grouped together

Reply:
I've broken them up more. Its multiline because it is javadoc, and it is javadoc because the name needs explanation - it's not `c:tridents`

Reply:
Ah forgot that javadoc is a thing, yes that makes sense in this case then